config/rootcompute/kube-1.24/clusterrole-kubernetes-maximal-permission-policy.yaml

```diff
@@ -3,24 +3,24 @@ kind: ClusterRole
 metadata:
   name: compute:apiexport:kubernetes:maximal-permission-policy
 rules:
-- apiGroups: [""]
-  resources: ["services"]
-  verbs: ["*"]
-- apiGroups: ["apps"]
-  resources: ["deployments"]
-  verbs: ["*"]
-- apiGroups: ["networking.k8s.io"]
-  resources: ["ingresses"]
-  verbs: ["*"]
-- apiGroups: [""]
-  resources: ["services/status"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["apps"]
-  resources: ["deployments/status"]
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["apps"]
-  resources: ["deployments/scale"]
-  verbs: ["get", "update", "patch"]
-- apiGroups: ["networking.k8s.io"]
-  resources: ["ingresses/status"]
-  verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
```

Comment:
Are these indentation changes on purpose ?

Reply:
my editor changes those automatically, I will try to keep the identation unchanged.